setup.py

```diff
@@ -230,7 +230,8 @@ def setup_package():
         FULLVERSION, GIT_REVISION = get_version_info()
         metadata['version'] = FULLVERSION
     else:
-        if len(sys.argv) >= 2 and sys.argv[1] == 'bdist_wheel':
+        if (len(sys.argv) >= 2 and sys.argv[1] == 'bdist_wheel') or (
+                    'develop' in sys.argv):
```

Comment:
Shouldn't this change to develop go up in line 219?
`sys.argv[1] in ('--help-commands', 'egg_info', '--version', 'clean', 'develop')`
I think we want to avoid entering the else condition on line 232 completely, since this is the block that imports numpy.